sc2ts/utils.py

```diff
@@ -1369,6 +1370,44 @@ def get_recombinant_samples(ts):
     return out
 
 
+def dereference_singleton_recombinants(ts, filter_nodes=False):
```

Comment:
"deference" is an odd choice here to me - I think of the C deference operator (*), which give you back the thing that a pointer points to.

How about "remove_singleton_recombinants"? You're just taking out the singleton recombinant sequences and their recombination nodes, right?

Reply:
So the choice of name is because by default we don't remove the nodes from the node table, but use `filter_nodes=False`. This is because some of the metadata refers to node IDs (I think), so I wanted to keep the ID mapping the same. 

I'm not particularly attached to "dereference", although "remove" implies to me that we are deleting them entirely. Any other suggestions for words to use?

Reply:
"Cull", "unreferenced", "disconnect", "detach"?

Reply:
detach?

Reply:
SGTM

Reply:
Done in latest push - good call @jeromekelleher, thanks.